CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+Development Process
+------------------
+
+Our default working branch is `develop`.  We do work by creating branches off `develop` for new features and bugfixes.  Any feature should include appropriate Cucumber acceptance tests and RSpec unit tests.  We try to avoid view and controller specs, and focus purely on unit tests at the model and service level where possible.  A bugfix may include an acceptance test depending on where the bug occurred, but fixing a bug should start with the creation of a test that replicates the bug, so that any bugfix submission will include an appropriate test as well as the fix itself.
+
+Each developer will usually work with a [fork](https://help.github.com/articles/fork-a-repo/) of the [main repository on Agile Ventures](https://github.com/AgileVentures/WebSiteOne). Before starting work on a new feature or bugfix, please ensure you have [synced your fork to upstream/develop](https://help.github.com/articles/syncing-a-fork/):
+
```

Comment:
I think that we need to ensure that new members understand what `upstream` is and how you set that remote. (I believe we had a section on that in a document on WSO. (http://www.agileventures.org/projects/websiteone/documents/how-to-submit-a-pull-request-on-github)

```
Assuming that you have the following local setup:
$ git remote -v
origin	   git@github.com:YourName/WebsiteOne.git (fetch)
origin	   git@github.com:YourName/WebsiteOne.git (push)
upstream   https://github.com/AgileVentures/WebsiteOne.git (fetch)
upstream   https://github.com/AgileVentures/WebsiteOne.git (push)
```